milatools/datasets/CIFAR10.py

```diff
@@ -0,0 +1,110 @@
+from milatools.datasets.cached import CachedDataset
```

Comment:
Usually the from imports are in the form `from milatools.datasets import cached` (https://google.github.io/styleguide/pyguide.html#22-imports)
to reduce namespace clashes, I'm not sure about the type style we want to have in this repo, @breuleux, what do you think?

Reply:
never had an issue with it, you can always rename with 'as' it in case of clashes